index.js

```diff
@@ -1,52 +1,57 @@
 #!/usr/bin/env node
-const path = require('path')
-const fs = require('fs-extra')
-const argv = require('minimist')(process.argv.slice(2))
+const path = require("path");
+const fs = require("fs-extra");
+const argv = require("minimist")(process.argv.slice(2));
 
 async function init() {
-  const targetDir = argv._[0] || '.'
-  const cwd = process.cwd()
-  const root = path.join(cwd, targetDir)
-  console.log(`Scaffolding project in ${root}...`)
+  const targetDir = argv._[0] || ".";
+  const cwd = process.cwd();
```

Comment:
Can you revert  double quotes and the semicolons of line end.

Reply:
Sorry, my editor auto-format code. Have recovered.